scripts/cluster/create_multinode_cluster.go

```diff
@@ -172,14 +166,29 @@ func ExtractMasterNodeInfo() error {
 		configs.Kube.ApiserverAdvertiseAddress,
 		configs.Kube.ApiserverPort,
 		configs.Kube.ApiserverToken,
-		configs.Kube.ApiserverTokenHash)
+		configs.Kube.ApiserverDiscoveryToken,
+		configs.Kube.ApiserverCertificateKey)
 	_, err = masterKeyYamlFile.WriteString(masterKeyYaml)
 	if !utils.CheckErrorWithTagAndMsg(err, "Failed to create masterKey.yaml with master node information!\n") {
 		return err
 	}
 
-	utils.SuccessPrintf("Join cluster from worker nodes with command: sudo kubeadm join %s:%s --token %s --discovery-token-ca-cert-hash %s\n",
-		configs.Kube.ApiserverAdvertiseAddress, configs.Kube.ApiserverPort, configs.Kube.ApiserverToken, configs.Kube.ApiserverTokenHash)
+	utils.SuccessPrintf("Join cluster from worker nodes as a new control plane node with command: "+
```

Comment:
Better to state that you need to join the cluster from backup control plane nodes, not the worker nodes.